packages/flutter/lib/src/cupertino/activity_indicator.dart

```diff
@@ -27,9 +27,13 @@ class CupertinoActivityIndicator extends StatefulWidget {
     Key key,
     this.animating = true,
     this.radius = _kDefaultIndicatorRadius,
+    this.progress = 1.0,
   }) : assert(animating != null),
        assert(radius != null),
        assert(radius > 0),
+       assert(progress != null),
+       assert(progress >= 0),
+       assert(progress <= 1),
```

Comment:
Nit: We usually include the `.0` for doubles, just to clarify that it is a double.

Reply:
Fixed. Also updated the `radius` variable above.